.travis.yml

```
@@ -0,0 +1,87 @@
+conditions: v1
+
+git:
+  depth: false
+
+language: python
+
+dist: trusty
```

Let's not test in the old VM
```suggestion
dist: xenial
```